data/districts.json

```diff
@@ -331,7 +331,7 @@
         ],
         "raj": [
             {
-                "name": "Bogura",
+                "name": "Bogra",
```

Comment:
Sorry, this is not correct fix, because the english spelling of newly changed to **Bogura**. Check [here](https://www.thedailystar.net/frontpage/five-districts-get-changed-english-spelling-1557259)